### DISCUSSION TITLE
How to toggle mobile classes on desktop?

### DISCUSSION THREAD

**Discussion by ikuuAlex** [tailwindlabs/tailwindcss#5259] [Help]
Hi there! I'm facing a small issue, which I can't figure out. How can I toggle SM: MD: XL: classes on a XL: screen without resizing? I want to preview some components as they would appear on mobile but visited on a desktop screen. Does anyone know the trick?

**Comment by simonswiss** [comment]
Hey!

If you don't want to resize anything (but just have buttons to toggle between the different screen sizes), you'd probably need to write some JS that removes all the classes starting with a responsive prefix for the mobile view, add the `sm:` classes for the `sm` breakpoint demo, etc.

This will likely be a decent amount of work though, and your best solution instead is probably to display your components inside an `iframe`, which gets resized when clicking on the different screen size buttons.

The iframe dimensions then become the "viewport", which allows you to display the component at different breakpoints without making the actual browser window change size 👍

**Reply by ikuuAlex** [reply]
Thank you for your reply. Any change you can tell how to show the content inside this iframe? I don't have a "scr" to link to, I want a component view inside this iframe. (I'm using nuxt)
